tests/integration/hdf5/test_image.py

```diff
@@ -15,8 +15,8 @@ def setUpContainer(self):
             data=np.ones((3, 3, 3)),
             unit='unit',
             external_file=['external_file'],
-            starting_frame=[1, 2, 3],
-            format='tiff',
+            starting_frame=[0],
+            format='external',
             timestamps=[1., 2., 3.],
             device=self.dev1,
         )
```

Comment:
I would also remove the `data` argument, as there should be either `external_file` or `data` specified but not both (right?), so far we only check if none of them is supplied. should I add a check for that too, or are there cases when this is correct?

Reply:
> I would also remove the `data` argument, as there should be either `external_file` or `data`

Agreed

> should I add a check for that too, or are there cases when this is correct?

I would add a check for this as well.